docs/csharp/roslyn/toc.md

```diff
@@ -0,0 +1,44 @@
+# [The .NET Compiler Platform SDK (Roslyn APIs)](index.md)
+## [Understand the compiler API model](compiler-api-model.md)
+## [Work with syntax](work-with-syntax.md)
+## [Work with semantics](work-with-semantics.md)
+## [Work with a workspace](work-with-workspace.md)
```

Comment:
Is documentation for Scripting APIs planned?

Reply:
Not yet. Do you want to add issues and help us design an outline?